### ISSUE TITLE
Taking a "Panorama" Photo destroys Auto-Upload

### ISSUE THREAD

**Issue by rwat1** [nextcloud/android#3053]
### Actual behaviour

Credit to @feutl  for discovering this per his https://github.com/nextcloud/android/issues/2979#issuecomment-422369040

**Taking a Panorama photo will break the Auto-upload.**

a) Will not upload Panorama photo
b) Will not upload Normal photos taken after Panorama
c) Deleting Panorama seems to make everything OK... (maybe, need confirmation, atleast for me)

### Expected behaviour
- It should Upload the Panorama and not affect future auto-uploads of normal photos.

 
### Steps to reproduce
1.  Take a Normal Picture (Confirm that Auto-Upload works properly)
2.  Take a Panorama Photo
3. See if it appears on Uploads Tab (try Forced Rescan)
----> Panorama does not appear
4. Take a Normal Photo
5. Neither Panorama nor Normal Photo appears on Upload queue tab (even after Forced Re-scan)
6. Delete Panorama, then other Normal photos seem to be detected and uploaded (atleast for me)

@feutl https://github.com/nextcloud/android/issues/2979#issuecomment-422369040 was the first to discover this, I was able to confirm and reproduce his observations.

### Environment data
Android version:   Android 8.0  

Device model:   Samsung S9+

Stock or customized system:   Stock system: Samsung Experience V9.0

Nextcloud app version:  Version 3.2.4

Nextcloud server version:  V13.0.6.1    (via NextCloudPi v0.59.15)

#### Web server error log
```
Insert your webserver log here
```How the Heck do I get these??
 
#### Nextcloud log (data/nextcloud.log)
```
How the Heck do I get these??
Insert your Nextcloud log here
```
How the Heck do I get these??

**NOTE:** Be super sure to remove sensitive data like passwords, note that everybody can look here! You can use the Issue Template application to prefill some of the required information: https://apps.nextcloud.com/apps/issuetemplate

**Comment by AndyScherzinger** [comment]
cc @tobiasKaminsky maybe related to #3049

**Comment by tobiasKaminsky** [comment]
I can try it later, but currently I am out of clue why the image is not uploaded.
#3049 is only about viewing the panorama picture.

**Comment by feutl** [comment]
If someone gives me a "custom build" apk or a dev apk or whatever writes appropriate log data needed to investigate the problem, let me know.

**Comment by tobiasKaminsky** [comment]
There is no custom build needed, please see https://github.com/nextcloud/android/blob/master/README.md#getting-debug-info-via-logcat

Thanks for trying to help!

**Comment by feutl** [comment]
I updated to 3.3.0 at I cannot recreate that situation.
Even the panorama pictures do get uploaded now. So whatever has changed, for now it is fixed. I report back as soon as it reappears.

**Comment by AndyScherzinger** [comment]
@rwat1 are you also experiencing the panos being auto uploaded with 3.3.0? Just asking since we didn't change anything related to this issue for 3.3.0 (a fix for displaying panos is part of 3.4.0)

**Comment by rwat1** [comment]
@AndyScherzinger @feutl @tobiasKaminsky 

I'm currently on v3.2.4... Now I will update to v3.0, and report back.

**Comment by rwat1** [comment]
@tobiasKaminsky @AndyScherzinger 

you guys can mark this as "fixed" since the V3.3.0 made it so the Panoramas can upload no problemo. 

Whatever you guys did with auto-upload, it made it much more Zippy and faster too :-) and fixed the Panorama and Textfile upload problem.

**Comment by AndyScherzinger** [comment]
@rwat1 thanks for the feedback, so closing it for now, please re-open in case the issue comes back. We actually didn't do much for auto upload except for the auto detection feature for new media folders so this is likely not related to the app itself :/
